lib/text_styles/text_style.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+final TextStyle fontSize36 = GoogleFonts.lato(
```

Comment:
remove numbers from naming, you can use XLarge, Large, Medium, Small, xSmall

Reply:
done